packages/block-editor/src/components/block-list/index.js

```diff
@@ -37,7 +37,7 @@ function BlockList( {
 	rootClientId,
 	isDraggable,
 	renderAppender,
-	__experimentalUIParts = {},
+	__experimentalUIParts = { hasPopover: true },
```

Comment:
Let's keep the argument pass-through agnostic of its defaults here and leave them to the places where they're actually used:

```suggestion
	__experimentalUIParts = {},
```